lib/safe_macros.c

```diff
@@ -726,6 +726,16 @@ int safe_mount(const char *file, const int lineno, void (*cleanup_fn)(void),
 {
 	int rval;
 
+	rval = mount(source, target, filesystemtype, mountflags, data);
+	if (rval == -1) {
```

Comment:
Shouldn't we do:
```
        if (!is_fuse(filesystemtype)) {
                tst_brkm(...)
        }
```

Here instead so that we carry on and get to the code that attempts to mount a fuse filesystem?

Reply:
No, because the is_fuse() check only shows filesystems that *might* use fuse. There's no way to know if fuse is really needed unless we try to mount normally and fail.

Reply:
Yes, but in a case that mounting normally fails we should try fuse, which we do not after you change, that is because the tst_brk() is used for fatal errors and causes the test to exit.

Reply:
Thanks, I've updated the commit. How does it look now?